Cargo.toml

```diff
@@ -42,11 +42,11 @@ lsp-types = { version = "0.60", features = ["proposed"] }
 lazy_static = "1"
 log = "0.4"
 num_cpus = "1"
-racer = { version = "2.1.37", default-features = false }
+racer = { version = "2.1.38", default-features = false }
 rand = "0.7"
 rayon = "1"
 rustc_tools_util = "0.2"
-rustfmt-nightly = { version = "1.4.20", git = "https://github.com/rust-lang/rustfmt", branch = "rustfmt-1.4.20" }
+rustfmt-nightly = { version = "1.4.21", git = "https://github.com/rust-lang/rustfmt", branch = "rustfmt-1.4.21" }
```

Comment:
I wanted to go ahead and open this so we could proceed if needed, though note this is still using the git reference

@topecongiro - any chance we could get rustfmt v1.4.21 published to crates.io (and v1.4.20 as well if you have time)